### ISSUE TITLE
Helm Repo: https://woodpecker-ci.org/index.yaml : 404 Not Found

### ISSUE THREAD

**Issue by Nold360** [woodpecker-ci/woodpecker#507]
### Component

other

### Describe the bug

When trying to add the Helm repo as described in the docs, it doesn't work:

```
$ helm repo add woodpecker https://woodpecker-ci.org/
Error: looks like "https://woodpecker-ci.org/" is not a valid chart repository or cannot be reached: failed to fetch https://woodpecker-ci.org/index.yaml : 404 Not Found
```

### System Info

```shell
Helm: version.BuildInfo{Version:"v3.5.4", GitCommit:"1b5edb69df3d3a08df77c9902dc17af864ff05d1", GitTreeState:"clean", GoVersion:"go1.15.11"}
```

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/woodpecker-ci/woodpecker/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://woodpecker-ci.org/docs/intro).
- [X] Check that there isn't [already an issue](https://github.com/woodpecker-ci/woodpecker/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A join our [Discord Chat Server](https://discord.gg/fcMQqSMXJy).

**Comment by anbraten** [comment]
We havent't releases a version based on the refactored code of the current master yet, but the documentation for this next version is already online. For now you have to use the helm-chart from the repo directly and wait for the 0.15.0 version to be releases to use the hosted chart. Sorry for the inconvenience which came from those huge refactorings.

**Comment by 6543** [comment]
`v0.15.0-rc1` got released manually based on #715 so it should work now & is automated for future tagging events
